impl/src/main/java/org/apache/myfaces/application/ViewIdSupport.java

```diff
@@ -482,14 +482,7 @@ protected String handleSuffixMapping(FacesContext context, String requestViewId)
             }
         }
 
-        // Otherwise, if a physical resource exists with the name requestViewId let that value be viewId.
-        if (isViewExistent(context,requestViewId))
-        {
-            return requestViewId;
-        }
-        
-        //Otherwise return null.
-        return null;
+        return requestViewId;
```

Comment:
Physical page doesn't exist, so it doesn't make sense to check if the view exists.  Hope this doesn't cause any errors, but the build was successful. I'll see what the TCK says.